tests/goth_tests/test_webapp.py

```diff
@@ -0,0 +1,51 @@
+"""Test Simple, db-enabled dApp."""
+import logging
+import os
+import signal
+from functools import partial
+
+import pytest
+import requests
+
+from goth.runner.probe import RequestorProbe
+
+from tests.goth_tests.utils import assert_app_states, assert_node_states
+
+logger = logging.getLogger("goth.test.test")
+
+
+@pytest.mark.parametrize("requestor_use_proxy", [False], indirect=True)
+async def test_simple_db_enabled_application(
+    requestor_use_proxy: bool,
+    goth_requestor_probe: RequestorProbe,
+) -> None:
+    """Test Simple, db-enabled dApp happy path."""
+    async with goth_requestor_probe.run_command_on_host(
+        "dapp-runner start --config configs/goth.yaml dapp-store/apps/webapp.yaml",
+        env=os.environ,
+    ) as (_cmd_task, cmd_monitor, process_monitor):
+        cmd_monitor.add_assertion(assert_app_states)
+        cmd_monitor.add_assertion(
+            partial(assert_node_states, "db"),
+            name='assert_node_states("db")',
+        )
+        cmd_monitor.add_assertion(
+            partial(assert_node_states, "http"),
+            name='assert_node_states("http")',
+        )
+        await cmd_monitor.wait_for_pattern(
+            ".*Starting app: Simple, db-enabled web application.*", timeout=10
+        )
+        await cmd_monitor.wait_for_pattern(".*Application started.", timeout=120)
+
+        url = "http://localhost:8080"  # TODO how to get this one
```

Comment:
As stderr shows up after `SIGINT` I cannot get this easily. I have two options:

1. Try to solve the no stderr issue again
2. Get this URL from data file (e.g. `/home/lucjan/.local/share/dapp_runner/36EGD4_20230125_14_47_32/data`)